q2_types/_alpha_diversity.py

```diff
@@ -0,0 +1,47 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os.path
+
+import pandas as pd
```

Comment:
Can you add pandas to setup.py? We'll already get it from depending on skbio but since we're explicitly importing pandas it's safer to include it in `install_requires`.